compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEInsertLoopsForVectorization.cpp

```diff
@@ -43,11 +44,31 @@ class AMDAIEInsertLoopsForVectorizationPass
     return nDims - firstDim;
   };
 
+  /// Tile the generic op using `tileSizes` and coalesce the generated tiling
+  /// loops in order to minimize the overhead of loop control/branch statements.
+  static void performTiling(IRRewriter &rewriter,
+                                     linalg::GenericOp genericOp,
+                                     SmallVector<int64_t> &tileSizes,
+                                     bool isTensorType) {
```

Comment:
Instead of passing this bool, I think you can derive it from the generic and simplify the function signature.